resources/js/interfaces/beatmapset-search-response.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+import BeatmapsetExtendedJson from 'interfaces/beatmapset-extended-json';
+
+type SortField = 'artist' | 'difficulty' | 'favourites' | 'plays' | 'ranked' | 'rating' | 'relevance' | 'title';
```

Comment:
combine with `sortNames` in https://github.com/ppy/osu-web/blob/ae31a218b68fbb2657de45999227348979553bad/resources/js/beatmaps/search-sort.tsx#L14 ?